apollo-compiler/src/test/graphql/com/example/fragment_with_inline_fragment/fragment/HeroDetails.java

```diff
@@ -279,30 +279,27 @@ public static Builder builder() {
     }
 
     public static class Fragments {
-      final Optional<DroidDetails> droidDetails;
+      final @NotNull DroidDetails droidDetails;
```

Comment:
as side effect of this PR, it is another issue addressed here. Named fragment `DroidDetails ` is defined on concrete type `Droid` that is type condition of inline fragment. So it's correct no `droidDetails ` is not optional